pkgs/by-name/co/construct/package.nix

```
@@ -0,0 +1,40 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "construct";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "Thomas-de-Bock";
+    repo = "construct";
+    rev = "${finalAttrs.version}";
```

Given that `version` is already a string and since the upstream release is already `0.1.0`, as opposed to some repos tagging with `v` as a prefix, e.g. `v0.1.0` the following should work too:

```suggestion
    rev = finalAttrs.version;
```